content/pages/info/foss.md

```diff
@@ -0,0 +1,15 @@
+---
+title: FOSS at RustFest
+---
+
+## Grants
+
+We give out travel and attendance grants to FOSS contributors. These often take the form of tickets and travel grants. Usually, these take the form of a campaign, such as the [libz blitz grant](https://blog.rustfest.eu/libz-blitz). We also assist project contributors to come to RustFest in other ways, for example by finding sponsors for them.
```

Comment:
@badboy is the campaign remark still true?

Reply:
Nope, right now we only have the scholarships, but not other grants.